api/lib/notifications/notificationController.js

```diff
@@ -145,6 +168,40 @@ class NotificationController {
       }
     })
   }
+
+  // users need to be registered individually per topic
+  subscribeUserToPushNotificationTopics(pushToken, topics) {
+    return Promise.all(
+      topics.map((topic) =>
+        firebaseAdmin.messaging().subscribeToTopic(pushToken, topic)
+      )
+    )
+  }
+
+  async sendPushNotification(notification) {
+    const topic = getTopic(notification)
+    const body = getMessageTitle(this.networks, notification)
+    // const image = getIcon(notification) // should we use an icon per notification or just use the Lunie logo?
```

Comment:
I like the Lunie logo. I think we should just stick to it, so people can immediately see that it is us talking to them. Plus the Lunie logo is great!